### PR TITLE
Preparing release v1.25.0

### PR DESCRIPTION
**Pull request by rubenvp8510**
Signed-off-by: Ruben Vargas <ruben.vp8510@gmail.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,19 @@
 Changes by Version
 ==================
 
+1.25.0 (2021-08-08)
+-------------------
+* Add support repetitive arguments to operand ([#1434](https://github.com/jaegertracing/jaeger-operator/pull/1434), [@rubenvp8510](https://github.com/rubenvp8510))
+* Allow TLS flags to be disabled ([#1440](https://github.com/jaegertracing/jaeger-operator/pull/1440), [@rubenvp8510](https://github.com/rubenvp8510))
+* Add gRPC port for jaeger-query into its service resource ([#1521](https://github.com/jaegertracing/jaeger-operator/pull/1521), [@rubenvp8510](https://github.com/rubenvp8510))
+* Update operatorhub script ([#1516](https://github.com/jaegertracing/jaeger-operator/pull/1516), [@rubenvp8510](https://github.com/rubenvp8510))
```
Comment (jpkrohling) [inline CHANGELOG.md:9]: This should be removed

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,19 @@
...
+* Sidecar not removed ([#1508](https://github.com/jaegertracing/jaeger-operator/pull/1508), [@mfz85](https://github.com/mfz85))
```
Comment (jpkrohling) [inline CHANGELOG.md:10]: Perhaps a better title?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,19 @@
...
+* Add support for GRPC storage plugin ([#1517](https://github.com/jaegertracing/jaeger-operator/pull/1517), [@pavolloffay](https://github.com/pavolloffay))
+* Fix overwritten default labels in label selectors of `Service` ([#1490](https://github.com/jaegertracing/jaeger-operator/pull/1490), [@rudeigerc](https://github.com/rudeigerc))
+* Add resources requests and limits to the operator ([#1515](https://github.com/jaegertracing/jaeger-operator/pull/1515), [@brunopadz](https://github.com/brunopadz))
+* Fix outdated contributing.md example ([#1511](https://github.com/jaegertracing/jaeger-operator/pull/1511), [@jan25](https://github.com/jan25))
```
Comment (jpkrohling) [inline CHANGELOG.md:14]: This should be removed

File: CHANGELOG.md (hunk 2 above)
Comment (rubenvp8510) [reply CHANGELOG.md]: I changed the title to a better one :)

**Review by jpkrohling — APPROVED** [review]

**Pull request merged**